jobs/payment-jobs/tasks/unpaid_invoice_notify_task.py

```diff
@@ -51,16 +51,28 @@ def _notify_for_ob(cls):  # pylint: disable=too-many-locals
             cast(InvoiceModel.created_on, Date) == notification_date.date()
         )).all()
         current_app.logger.debug(f'Found {len(notification_pending_invoices)} invoices to notify admins.')
+        invoice_by_payment_accounts = {}
+
         for invoice in notification_pending_invoices:
+            if invoice.payment_account_id in invoice_by_payment_accounts:
+                invoice_by_payment_accounts[invoice.payment_account_id].total = \
```

Comment:
Here it's changing the actual invoice total value, and db would get updated. Ideally nothing on the model instance should be changed, for logic purpose.